main.js

```diff
@@ -0,0 +1,32 @@
+const navItems = document.querySelectorAll("nav ul li");
+console.log(navItems);
+
+for(let el of navItems){
+    if(el.textContent.includes("All Posts")){
+        el.remove();
+    }
+}
+
+const spanEls = document.querySelectorAll("p span");
+console.log(spanEls);
+
+for(let el of spanEls){
+    if(el.textContent.includes("Read Time")){
+        el.remove();
+    }
+}
```

Comment:
```suggestion
let firstSpan = document.querySelector("span")
firstSpan.remove()
OR
document.querySelector("span").remove();
```
This one can also be done in one line, because all of the `<span>` elements happen to only be used for "Read Time". Again, if that were not the case, and `<span>` tags were being used in other places, the for loop would be necessary.